[Doc] linalg.ldl_factor: render the Shape of tensor A

isdanni opened the pull request:
Summary: Fix: #96864

Test Plan: Please see GitHub tests.

lezcano reviewed (changes requested):
Please, see the format that's used in the rest of linalg. In particular, we avoid using `:math:` as it takes quite a long time to render. Use single backticks instead.

lezcano commented on torch/linalg/__init__.py:
```diff
@@ -871,7 +871,8 @@
-    A (Tensor): tensor of shape (*, n, n) where * is zero or more batch dimensions consisting of symmetric or Hermitian matrices.
+    A (Tensor): tensor of shape `(*, n, n)` where `*` is zero or more batch dimensions
+                    consisting of symmetric or Hermitian matrices.
                     `(*, n, n)` where `*` is one or more batch dimensions.
```
Feel free to delete this line as well, as it's repeated. Same in the other function.

lezcano replied:
Also, this line has extra indentation. Same in the other one

lezcano reviewed (approved):
Thank you!

isdanni commented:
@lezcano @IvanYashchuk any plans to merge this? :)

lezcano commented:
You can merge it yourself writing `@pytorchbot merge` in a comment  :)

isdanni commented:
@pytorchbot merge

isdanni commented:
@pytorchbot merge -r

isdanni commented:
@pytorchbot merge -r

isdanni commented:
@pytorchbot merge -f

isdanni commented:
@pytorchbot merge -f "test failure unrelated"

isdanni committed:
[doc] linalg.ldl_factor: render the Shape of tensor A (#99777)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/99777

Differential Revision: D45204660

fbshipit-source-id: c797c01364df96b0b71073244527f45c8ddba3ea

lezcano commented:
Since you imported this, you should import the rebased branch onto phabricator as well, I believe.

isdanni commented:
@pytorchbot merge
